doc/classes/YSort.xml

```diff
@@ -1,20 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="YSort" inherits="Node2D" version="4.0">
 	<brief_description>
-		Sort all child nodes based on their Y positions.
+		[i]Deprecated.[/i] Sort all child nodes based on their Y positions. This has been replaced by the [code]y_sort_enabled[/code] property in [Node2D].
 	</brief_description>
 	<description>
-		Sort all child nodes based on their Y positions. The child node must inherit from [CanvasItem] for it to be sorted. Nodes that have a higher Y position will be drawn later, so they will appear on top of nodes that have a lower Y position.
+		[i]Deprecated.[/i] Sort all child nodes based on their Y positions. The child node must inherit from [CanvasItem] for it to be sorted. Nodes that have a higher Y position will be drawn later, so they will appear on top of nodes that have a lower Y position.
 		Nesting of YSort nodes is possible. Children YSort nodes will be sorted in the same space as the parent YSort, allowing to better organize a scene or divide it in multiple ones, yet keep the unique sorting.
+		This functionality has been replaced by the [code]y_sort_enabled[/code] property in [Node2D].
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
-		<member name="sort_enabled" type="bool" setter="set_sort_enabled" getter="is_sort_enabled" default="true">
-			If [code]true[/code], child nodes are sorted, otherwise sorting is disabled.
-		</member>
```

Comment:
This should be preserved if we're not outright remove the node, so existing scripts still work (and thus it should change `Node2D.y_sort_enabled` when toggled).

Reply:
Fixed, I also kept both functions set_sort_enabled and is_sort_enabled in order to ensure full compatibility while it is deprecated. They just immediately call set_y_sort_enabled and is_y_sort_enabled respectively.